app/API/V1/Repositories/RoleRepository.php

```diff
@@ -83,6 +84,29 @@ public function removePermissions(array $rolePermissions, bool $delete=false):vo
         }
     }
 
+    /**
+     * Set user's default role
+     *
+     * @param User $user
+     * @throws \Doctrine\DBAL\ConnectionException
+     */
+    public function setUserPermissions(User $user):void
```

Comment:
Should be a method on the user repo that takes a list of roles and assigns them to a user, so it can be used universally. Would be called: addRoles